### PR TITLE
DOCS-6388: Changes to page about uploading monitors

### PR DESCRIPTION
**Pull request by rddheath**
I created this PR in response to this thread in the #functional-monitoring channel on Slack: https://mulesoft.slack.com/archives/C8XR3MY4T/p1569856669014500

### CODE REVIEW

**Review by diegor — APPROVED** [review]

File: modules/ROOT/pages/afm-upload-monitor.adoc [hunk 1]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-upload-monitor.adoc:5]: INSTRUCTIONS for edit:

Only the paragraph at line 6 is in need of your skills. The rest I've not changed, and I know not why GitHub reports a change at line 54.

File: modules/ROOT/pages/afm-upload-monitor.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
...
 
-You can upload a suite of tests that you have written and want to run as a monitor, viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring.
+You can upload a suite of tests that you have written and want to run as a monitor, viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring. When you create monitors manually, you can create more complex test cases by using features such as the DataWeave expression language and context variables for saving values between calls to endpoints.
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-upload-monitor.adoc]: This is a bit unwieldy, given the three user actions already in the beginning of the sentence: "viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring." I crave minimalism; it's my simple mind!
```suggestion
To create your own monitor, uploading a suite of tests that you have written to obtain more specific or complex results. You can then run the tests and view the results in the *Functional Monitoring* section of Anypoint Monitoring. By creating monitors manually, you can create more complex test cases by using features such as the DataWeave expression language and context variables for saving values between calls to endpoints. 
```
Do we need to say anything about the alternative to creating monitors manually? IS there an alternative: default monitoring operations in Anypoint Monitoring? Have these already been explained? Should we begin this text with the advantage, something like "To enable customized and complex monitoring, you can create your own suite of tests, upload it, run it, and view the results..."?

File: modules/ROOT/pages/afm-upload-monitor.adoc [hunk 3]
```diff
@@ -51,4 +51,4 @@ If you want to delete the monitor, delete all versions of it in the *Tests* sect
 
 == See also
 
-* xref:bat-top.adoc[API Functional Monitoring with the BAT CLI]
\ No newline at end of file
+* xref:bat-top.adoc[API Functional Monitoring with the BAT CLI]
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-upload-monitor.adoc:54]: Silly GitHub!

File: modules/ROOT/pages/afm-upload-monitor.adoc [hunk 4]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-You can upload a suite of tests that you have written and want to run as a monitor, viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring.
+You can upload a suite of tests that you have written and want to run as a monitor, viewing the results the information displays in the *Functional Monitoring* section of Anypoint Monitoring. When you create monitors manually, you can create more complex test cases by using features such as the DataWeave expression language and context variables for saving values between calls to endpoints. 
 
 
 == Before you begin
```
Comment (kbostrom) [inline modules/ROOT/pages/afm-upload-monitor.adoc]:
```suggestion
== Before You Begin
```
Most of your peers are using Prerequisites. 
I am trying to get templated, architectural standardization, but failing so far....

**Review by kbostrom — APPROVED** [review]
You are the lucky recipient of the fruits of a well-deserved but temporary lull in the editorial madness that is my queue. As a result, I have engaged in a lengthier brainstorming edit than these two sentences really deserve.... Fair warning.

**Pull request merged**